src/app/dashboard/clients/page.tsx

```diff
@@ -1,7 +1,29 @@
-export default function Clients() {
+import { ClientTable } from "@/components/clientTable/ClientTable";
+import { columns, Client } from "@/components/clientTable/columns";
+
+async function fetchData(): Promise<Client[]> {
```

Comment:
For clarity:
```suggestion
async function fetchClients(): Promise<Client[]> {
```

Reply:
I'd also recommend moving this function to an external `src/lib/data/clients.ts` file to keep server data fetching in one place (with the other lib files).